schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-core-distribution.xsl

```diff
@@ -0,0 +1,443 @@
+              Range:	xsd:decimal
+              Usage note:	If the dataset is an image or grid this should correspond to the spacing of items. For other kinds of spatial datasets, this property will usually indicate the smallest distance between items in the dataset.
+              Usage note:	Alternative spatial resolutions might be provided as different dataset distributions
+              -->
+              <xsl:apply-templates mode="iso19115-3-to-dcat"
+                                   select="ancestor::mdb:MD_Metadata/mdb:identificationInfo/*/mri:spatialResolution/*/mri:distance
+                                          |ancestor::mdb:MD_Metadata/mdb:identificationInfo/*/mri:temporalResolution/*"/>
+
+              <!--
+              RDF Property:	odrl:hasPolicy
+              Definition:	An ODRL conformant policy expressing the rights associated with the distribution.
+              Range:	odrl:Policy
+              Usage note:	Information about rights expressed as an ODRL policy [ODRL-MODEL] using the ODRL vocabulary [ODRL-VOCAB] MAY be provided for the distribution. See also guidance at 9. License and rights statements.
+
+              Not supported
+              -->
+
+              <!--
+              RDF Property:	dcterms:conformsTo
+              Definition:	An established standard to which the distribution conforms.
+              Range:	dcterms:Standard (A basis for comparison; a reference point against which other things can be evaluated.)
+              Usage note:	This property SHOULD be used to indicate the model, schema, ontology, view or profile that this representation of a dataset conforms to. This is (generally) a complementary concern to the media-type or format.
+              -->
+              <xsl:apply-templates mode="iso19115-3-to-dcat"
+                                   select="ancestor::mdb:MD_Metadata/mdb:dataQualityInfo/*/mdq:report/*/mdq:result[mdq:DQ_ConformanceResult and mdq:DQ_ConformanceResult/mdq:pass/*/text() = 'true']"/>
+            </xsl:if>
+
+            <xsl:copy-of select="$additionalProperties"/>
+          </dcat:Distribution>
+        </dcat:distribution>
+      </xsl:otherwise>
+    </xsl:choose>
+  </xsl:template>
+
+
+  <xsl:template mode="iso19115-3-to-dcat-distribution"
+                match="mrd:distributionFormat/*/mrd:formatSpecificationCitation">
+    <xsl:call-template name="rdf-format-as-mediatype">
+      <xsl:with-param name="format" select="*/cit:title/*/text()"/>
+    </xsl:call-template>
+  </xsl:template>
+
+
+  <xsl:template mode="iso19115-3-to-dcat-distribution"
+                match="mrd:distributionFormat/*/mrd:fileDecompressionTechnique">
+    <xsl:call-template name="rdf-format-as-mediatype">
+      <xsl:with-param name="elementName" select="'dcat:compressFormat'"/>
+      <xsl:with-param name="format" select="*/text()"/>
+    </xsl:call-template>
+  </xsl:template>
```

Comment:
Unclear as doesn’t seem to work, also apparently it seems the same value assigned to all the online resources?

Reply:
Yes this can be challenging as DCAT distribution contains a number of properties that ISO affect to the resource. So it could work fine if we have only one download link in a record or could be encoded using different distribution block but it is not the case usually.

There is a test for example
https://github.com/SPW-DIG/metawal-core-geonetwork/blob/dcat/services/src/test/resources/org/fao/geonet/api/records/formatters/iso19115-3.2018-dcat-dataset.xml#L1254-L1294
is converted to
https://github.com/SPW-DIG/metawal-core-geonetwork/blob/dcat/services/src/test/resources/org/fao/geonet/api/records/formatters/iso19115-3.2018-eu-dcat-ap-dataset-core.rdf#L554-L570